src/components/Courses/Courses.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react';
+import { useState } from 'react';
+import { CourseCard } from './Components/CourseCard/CourseCard';
+import { SearchBar } from './Components/SearchBar/SearchBar';
+import { EmptyCourseList } from '../EmptyCourseList/EmptyCourseList';
+import { CourseDataProps, CoursesProps } from 'helpers/Types';
+
+export const Courses = (props: CoursesProps) => {
+  const { courseData, authorData, setCurrentCourse } = props;
+
+  const [searchTerm, setSearchTerm] = useState('');
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setSearchTerm(e.target.value);
+  };
+  return (
+    <div className='main-container'>
+      {courseData.length >= 1 ? (
+        <Fragment>
+          <SearchBar>
+            <input
+              style={{
+                width: '564px',
+                height: '50px',
+                fontSize: '20px',
+              }}
+              onChange={handleChange}
+              value={searchTerm}
+              type='text'
+              placeholder='Input text'
+            />
+          </SearchBar>
+
+          {courseData
+            .filter((item: CourseDataProps) => {
+              return searchTerm.toLowerCase() === ''
+                ? item
+                : item.title.toLowerCase().includes(searchTerm) ||
```

Comment:
You need to lowercase both the search query and the item when you compare, or else the query won't be found if it isn't typed in lowercase.

![image](https://github.com/elenapetra/react-app/assets/145835962/aed813a0-f76b-4a9d-8c9a-61a9673fab7c)

Reply:
Fixed